### ISSUE TITLE
Click on toolbar area between buttons hides the toolbar

### ISSUE THREAD

**Issue by mmotyczynska** [ckeditor/ckeditor5#12172]
## 📝 Provide detailed reproduction steps (if any)

1. In ClassicEditor scroll down so the toolbar is not visible (in balloon toolbar it's not necessary).
2. Make a selection in the editor so the toolbar appears.
3. Click the toolbar somewhere between buttons.

This should be useful in case someone misses the button and accidentally clicks the toolbar instead of a button.

### ✔️ Expected result

The toolbar is still visible.

### ❌ Actual result

The toolbar hides.

https://user-images.githubusercontent.com/95343030/181464786-be290e0e-abff-44f5-8781-d38fa34ee351.mov

### ❓ Possible solution

When someone clicks the toolbar instead of a button we could move the focus to the first item in the toolbar.

## 📃 Other details

* Browser: …
* OS: …
* First affected CKEditor version: …
* Installed CKEditor plugins: …

---

If you'd like to see this fixed sooner, add a 👍 reaction to this post.

**Comment by tony1377** [comment]
Any idea of an ETA on this?

**Comment by mlewand** [comment]
@tony1377 not yet. Please add 👍 reaction to the main post to show your support for it.
